### PR TITLE
Update Batch registration helper script

### PR DESCRIPTION
**Pull request by bxmmm1**
- Better UX in the registration script
- Skip non `.json` files
- Fix compiler warnings

### CODE REVIEW

File: script/BatchRegisterValidator.s.sol [hunk 1]
```diff
@@ -120,4 +160,15 @@ contract BatchRegisterValidator is Script {
 
         return Permit({ deadline: deadline, amount: amount, v: v, r: r, s: s });
     }
+
+    function isJsonFile(string calldata str) external pure returns (bool) {
+        uint256 length = bytes(str).length;
+        if (length < 5) {
```
Comment (shayanb) [inline script/BatchRegisterValidator.s.sol:166]: Why length < 5 for file?  what are you checking here?

File: script/BatchRegisterValidator.s.sol [hunk 2]
```diff
@@ -54,14 +60,37 @@ contract BatchRegisterValidator is Script {
         uint256 vtAmount = vm.promptUint("Enter the VT amount per validator (28 is minimum):");
```
Comment (shayanb) [inline script/BatchRegisterValidator.s.sol:60]: just something I forgot to push on my last commit: remove the `:` from the end, promtUint already prints one.

**Review by shayanb — APPROVED** [review]

File: script/BatchRegisterValidator.s.sol (hunk 1 above)
Comment (shayanb) [reply script/BatchRegisterValidator.s.sol:166]: ah it's the file extension `.json` . let's add this to the docs (make sure it's part of the commands to include .json

**Comment by bxmmm1** [comment]
I'm working on an updated version of the script that will use `coral-cli` to generate new key, and register it right away

**Pull request merged**